src/components/autocomplete/js/autocompleteDirective.js

```diff
@@ -222,7 +222,8 @@ function MdAutocomplete () {
                   aria-autocomplete="list"\
                   aria-haspopup="true"\
                   aria-activedescendant=""\
-                  aria-expanded="{{!$mdAutocompleteCtrl.hidden}}"/>\
```

Comment:
String construction using `\` is denigrated in Google Style docs. 
We should be using string concatenation; e.g. [tabsDirective.js](https://github.com/angular/material/blob/master/src/components/tabs/js/tabsDirective.js#L98-L119)